tock/tock/static/sass/_header.scss

```diff
@@ -0,0 +1,43 @@
+// To mimic behavior of primary nav dropdown
+.usa-nav__secondary {
+  button,
+  a {
+    padding: 1rem;
+    display: inline-block;
+  }
+  
+  .usa-accordion__button {
+    background-color: transparent;
+    width: auto;
+    font-weight: normal; 
+    color: #71767a;
```

Comment:
Do we have (or should we set) a var for the color?

Reply:
yasssss